src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/PackageDeclarationCheck.java

```diff
@@ -100,9 +100,6 @@
      */
     public static final String MSG_KEY_MISMATCH = "mismatch.package.directory";
 
-    /** Line number used to log violation when no AST nodes are present in file. */
-    private static final int DEFAULT_LINE_NUMBER = 1;
```

Comment:
maybe it's worth keeping this in the code for the comment explaining it?

Reply:
I thought why to use an extra variable, so I removed this.

Reply:
I don’t think it’s a big deal, but probably lean toward keeping the variable and comment in case it makes the code more readable for someone looking at it.